src/site/_includes/layout.njk

```diff
@@ -13,11 +13,12 @@
       <meta name="offline" content="true" />
     {%- endif %}
 
-    {% if not nometa %}
+    {% if nometa %}
```

Comment:
Would you mind adding a comment to this block to explain what's happening. Essentially that for most posts we use Meta to give them their metadata but nometa is used by the sw partial and we want to put a token in there that we can replace at runtime.